scripts/merge-locales.js

```
@@ -0,0 +1,86 @@
+
+// Write JSON file
+const writeJsonFile = (filePath, data) => {
+  fs.writeFileSync(filePath, JSON.stringify(data, null, 2), 'utf-8');
+};
+
+// Deep merge two objects
+const deepMerge = (target, source) => {
+  const output = { ...target };
+
+  for (const key in source) {
+    if (source.hasOwnProperty(key)) {
```

Biome (1.9.4)

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)